extensions/amp-story/0.1/system-layer.js

```diff
@@ -222,6 +222,12 @@ export class SystemLayer {
         this.onUnmuteAudioClick_(e);
       }
     });
+
+    this.win_.addEventListener('keyup', e => {
+      if (!this.closeBookendBtn_.hasAttribute('hidden') && e.key == 'Escape') {
+        this.onCloseBookendClick_(e);
```

Comment:
nit: maybe rename this function to something generic to both click and keypress like `closeBookend`?  @alanorozco might have better ideas for naming

Reply:
Done